src/requirements/data/colleges/as.ts

```diff
@@ -146,11 +149,11 @@ const casRequirements: readonly CollegeOrMajorRequirement[] = [
       'Five Arts & Sciences courses of 3 or more credits from at least 4 of the following social sciences, humanities, and arts categories:',
     source: 'https://as.cornell.edu/degree-requirements',
     checker: [
-      (course: Course): boolean => course.catalogDistr?.includes('(CA-AS)') ?? false,
-      (course: Course): boolean => course.catalogDistr?.includes('(HA-AS)') ?? false,
-      (course: Course): boolean => course.catalogDistr?.includes('(KCM-AS)') ?? false,
-      (course: Course): boolean => course.catalogDistr?.includes('(LA-AS)') ?? false,
-      (course: Course): boolean => course.catalogDistr?.includes('(SBA-AS)') ?? false,
+      (course: Course): boolean => course.catalogDistr?.includes('CA-AS') ?? false,
+      (course: Course): boolean => course.catalogDistr?.includes('HA-AS') ?? false,
+      (course: Course): boolean => course.catalogDistr?.includes('KCM-AS') ?? false,
+      (course: Course): boolean => course.catalogDistr?.includes('LA-AS') ?? false,
+      (course: Course): boolean => course.catalogDistr?.includes('SBA-AS') ?? false,
```

Comment:
Due to the additional of new categories, there might be multiple items between the parenthesis.
e.g. [ASIAN 2212](https://classes.cornell.edu/browse/roster/SP21/class/ASIAN/2212) originally only has `(CA-AS)`, now it's `(CA-AS, ALC-AS, HST-AS)`. Therefore, we should not include parenthesis in the matching. Similar changes are made elsewhere.